tools/proteomiqon_labeledproteinquantification/aggregationparams.xml

```diff
@@ -0,0 +1,129 @@
+                </when>
+            </conditional>
+            <conditional name="LabeledTransformHeavyCond">
+                <param name="Type" type="select" label="Heavy">
+                    <option value="None">None</option>
+                    <option value="Log2">Log2</option>
+                    <option value="Substract">Substract</option>
+                    <option value="Add">Add</option>
+                    <option value="DivideBy">DivideBy</option>
+                    <option value="MultiplyBy">MultiplyBy</option>
+                </param>
+                <when value="None">
+                </when>
+                <when value="Log2">
+                </when>
+                <when value="Substract">
+                    <param name="Value" type="float" value="0.0" optional="true" label="Value" />
+                </when>
+                <when value="Add">
+                    <param name="Value" type="float" value="0.0" optional="true" label="Value" />
+                </when>
+                <when value="DivideBy">
+                    <param name="Value" type="float" value="0.0" optional="true" label="Value" />
+                </when>
+                <when value="MultiplyBy">
+                    <param name="Value" type="float" value="0.0" optional="true" label="Value" />
+                </when>
+            </conditional>
+            <conditional name="LabeledTransformRatioCond">
+                <param name="Type" type="select" label="Ratio">
+                    <option value="None">None</option>
+                    <option value="Log2">Log2</option>
+                    <option value="Substract">Substract</option>
+                    <option value="Add">Add</option>
+                    <option value="DivideBy">DivideBy</option>
+                    <option value="MultiplyBy">MultiplyBy</option>
+                </param>
+                <when value="None">
+                </when>
+                <when value="Log2">
+                </when>
+                <when value="Substract">
+                    <param name="Value" type="float" value="0.0" optional="true" label="Value" />
+                </when>
+                <when value="Add">
+                    <param name="Value" type="float" value="0.0" optional="true" label="Value" />
+                </when>
+                <when value="DivideBy">
+                    <param name="Value" type="float" value="0.0" optional="true" label="Value" />
+                </when>
+                <when value="MultiplyBy">
+                    <param name="Value" type="float" value="0.0" optional="true" label="Value" />
+                </when>
+            </conditional>                                    
+        </section>
+        <section name="LabeledSingleFilters" title="Labeled Single filters">
+            <section name="Light" title="Light">
+                <param name="BiggerThan" type="float" value="" optional="true" label="Is bigger than" />
```

Comment:
We should add a widget to Galaxy that supports those kinds of elements :)

Reply:
Yeah, that would come in handy!